commands/container-registry/name/names.go

```diff
@@ -0,0 +1,50 @@
+package name
+
+import (
+	"context"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/core"
+)
+
+func RegNamesCmd() *core.Command {
+	cmd := core.NewCommand(
+		context.TODO(), nil, core.CommandBuilder{
+			Namespace:  "container-registry",
+			Resource:   "registry",
+			Verb:       "names",
+			Aliases:    []string{"check", "name", "n"},
+			ShortDesc:  "Check if a Registry Name is available",
+			LongDesc:   "Check if a Registry Name is available",
+			Example:    "ionosctl container-registry name",
+			PreCmdRun:  PreCmdCheck,
+			CmdRun:     CmdCheck,
+			InitClient: true,
+		},
+	)
+
+	cmd.AddStringFlag("name", "", "", "Name to check availability for", core.RequiredFlagOption())
+	return cmd
+}
+
+func CmdCheck(c *core.CommandConfig) error {
+	name, err := c.Command.Command.Flags().GetString("name")
+	if err != nil {
+		return err
+	}
+	res, _ := c.ContainerRegistryServices.Name().Head(name)
+	if res.StatusCode == 404 {
+		return c.Printer.Print("Name is available.")
+	}
+	if res.StatusCode == 400 {
+		return c.Printer.Print("Bad request.")
```

Comment:
what exactly is wrong with the request? perhaps print the error in this case, or some other API message to give a hint of what's wrong

```
 % ionosctl cr names --name hi
Bad request.
```